en/guide/configuration.md

```diff
@@ -51,40 +51,40 @@ This option lets you to define all default meta tags for your application.
 
 This option lets you to customize the loading component which is loaded by default with Nuxt.js.
 
-[Documentation about loading integration](/api/configuration-loading)
+[Documentation about `loading` integration](/api/configuration-loading)
 
 ### modules
 
-This option lets you add nuxt modules to your project.
+This option lets you add Nuxt modules to your project.
 
-[Documentation about modules integration](/api/configuration-modules)
+[Documentation about `modules` integration](/api/configuration-modules)
 
 ### plugins
 
-This option lets you to define JavaScript plugins to be run before instantiating the root vue.js application.
+This option lets you to define JavaScript plugins to be run before instantiating the root Vue.js Application.
 
-[Documentation about plugins integration](/api/configuration-plugins)
+[Documentation about `plugins` integration](/api/configuration-plugins)
 
 ### rootDir
 
-This option lets you define the workspace of your nuxt.js application.
+This option lets you define the workspace of your Nuxt.js Application.
 
-[Documentation about rootDir integration](/api/configuration-rootdir)
+[Documentation about `rootDir` integration](/api/configuration-rootdir)
 
 ### router
 
-This option lets you to overwrite the default Nuxt.js configuration of vue-router.
+This option lets you to overwrite the default Nuxt.js configuration of Vue Router.
```

Comment:
Use the product name for Vue Router (not package name) for more conistency with Nuxt.js.